web/api/v2/verify/index.ts

```diff
@@ -19,7 +19,7 @@ const schema = yup.object({
     .string()
     .strict()
     .nonNullable()
-    .required("This attribute is required."),
+    .defined("This attribute is required."),
```

Comment:
I think I'd prefer to keep this as `.required()` to prevent proof overlap between Incognito Actions and Sign In with World ID, pending approval from @andy-t-wang -- is there any way for us to tell if there are apps using `action: ""` for incognito actions?

If we do move forward with preventing empty actions, we should also update IDKit to make `action` a required parameter.

Reply:
Updated.